tests/callbacks/test_progress_bar.py

```diff
@@ -146,8 +146,6 @@ def test_progress_bar_fast_dev_run(tmpdir):
     assert 1 == progress_bar.total_train_batches
     # total val batches are known only after val dataloaders have reloaded
 
-    trainer.fit(model)
-
```

Comment:
why remove this?

Reply:
When the User would use the `fast_dev_run=True` flag, they won’t be able to call `.fit()` twice or more, as it would reach the `max_steps`.

Reply:
oh okay, it's on fast_dev_run.. didn't see that.